components/filter_image_resolution/fondant_component.yaml

```diff
@@ -1,14 +1,14 @@
 name: Filter image resolution
 description: Component that filters images based on minimum size and max aspect ratio
-image: ghcr.io/ml6team/filter_image_resolution:latest
+image: ghcr.io/ml6team/filter_image_resolution:f3f3925b8e8f634e2978e5c7fcefa72c53baba7c
 
 consumes:
   image:
     fields:
       width:
-        type: int16
+        type: int64
```

Comment:
int16 maxes out at 32.767 which is why you may be having issues if your image's width was 36.000 
However, I think this should be resolved if you move to `int32`